cwms-data-api/src/main/java/cwms/cda/api/TimeSeriesController.java

```diff
@@ -1,46 +1,7 @@
 package cwms.cda.api;
 
 import static com.codahale.metrics.MetricRegistry.name;
-
-import static cwms.cda.api.Controllers.ACCEPT;
```

Comment:
Please review the checkstyle.xml file, given the number of elements it would be reasonable to add Controllers as an exception for `import <pkg>.*;` but otherwise we don't do that.

Reply:
Done

Reply:
still have some `java.util` in there

Reply:
done